prims/prims.c

```diff
@@ -0,0 +1,48 @@
+#include <stdio.h>
+#include <string.h>
+
+# define SIZE 5
+# define INFINITY 1000
+
+// This function finds the minimal spanning tree by Prim's Algorithm
+void prims(int G[SIZE][SIZE], int *parent) {
+    int select[SIZE], i, j, k;
+    int min_dist = INFINITY;
```

Comment:
To solve this issue remove this line and on line 16 make it `int min_dist = INFINITY`